src/utils.ts

```diff
@@ -168,20 +169,21 @@ export function maybeAddComment(
       content = prefix + content;
     }
 
-    const lines = content.split('\n').map((l) => l.replace(/^ /, '').replace(/\n/, ''));
-
-    if (deprecated) {
-      lines.push('');
-      lines.push('@deprecated');
-    }
-
-    let comment: Code;
-    if (lines.length === 1) {
-      comment = code`/** ${content} */`;
-    } else {
-      comment = code`/**\n * ${lines.join('\n * ')}\n */`;
-    }
+    lines = content.split('\n').map((l) => l.replace(/^ /, '').replace(/\n/, ''));
+  }
+  // Deprecated comment should be added even if no other comment was added
+  if (deprecated) {
+    lines.push('');
```

Comment:
Wdyt about only pushing this empty string if lines is non-empty? 

That would make `testField` look like:

```
  /** @deprecated */
  testField: string;
```

Reply:
Yeah I thought about it as well at first. Will add it now